Test/DafnyTestGeneration/TestGeneration.dfy.expect

```diff
@@ -0,0 +1,16 @@
+include "TestGeneration.dfy"
+module TestGenerationUnitTests {
+import M
+method test0(v0:M.Value) returns (r0:int)  modifies v0 {
+v0.v := -INT;
```

Comment:
This doesn't look right - was `INT` supposed to be substituted?

Reply:
It is, since there is no way of telling which particular integer the solver will come up with. I am actually modifying the test right now to execute this dafny code - will let you know once I succeed.